webapp/client/client.jsx

```diff
@@ -1397,7 +1397,7 @@ export default class Client {
             end(this.handleResponse.bind(this, 'updateLastViewedAt', success, error));
     }
 
-    // SCHEDULED FOR DEPRECATION IN 3.8 - use viewChannel instead
+    // SCHEDULED FOR DEPRECATION IN 3.8
```

Comment:
@jwilander FYI I think you forgot to remove the `time` parameter for the `viewChannel` method in this file